package.json

```diff
@@ -26,40 +26,30 @@
   ],
   "dependencies": {},
   "devDependencies": {
-    "@types/moment": "^2.13.0",
-    "@types/chai": "^3.4.34",
-    "@types/es6-shim": "^0.31.32",
-    "@types/mocha": "^2.2.33",
-    "@types/node": "0.0.2",
-    "@types/sinon": "^2.2.2",
-    "chai": "^4.2.0",
-    "chai-as-promised": "^7.1.1",
-    "del": "^2.2.1",
-    "es6-shim": "^0.35.1",
-    "gulp": "^3.9.1",
-    "gulp-file": "^0.3.0",
-    "gulp-istanbul": "^1.0.0",
-    "gulp-mocha": "^3.0.1",
-    "gulp-rename": "^1.2.2",
-    "gulp-replace": "^0.5.4",
-    "gulp-shell": "^0.5.1",
-    "gulp-sourcemaps": "^1.6.0",
-    "gulp-tslint": "^8.1.3",
-    "gulp-typescript": "^3.1.3",
-    "gulp-uglify": "^2.0.0",
-    "gulpclass": "^0.1.1",
-    "mocha": "^3.2.0",
-    "moment": "2.14.1",
-    "reflect-metadata": "^0.1.3",
-    "remap-istanbul": "^0.7.0",
-    "sinon": "^1.17.4",
-    "sinon-chai": "^2.8.0",
-    "tslint": "^4.0.2",
-    "tslint-stylish": "^2.1.0-beta",
-    "typescript": "^2.0.10"
+    "@types/jest": "25.1.5",
+    "@types/node": "13.11.0",
+    "@typescript-eslint/eslint-plugin": "2.26.0",
+    "@typescript-eslint/parser": "2.26.0",
+    "copyfiles": "2.2.0",
+    "dayjs": "1.8.23",
+    "eslint": "6.8.0",
+    "eslint-plugin-jest": "23.8.2",
+    "jest": "25.2.7",
+    "json": "9.0.6",
+    "reflect-metadata": "0.1.13",
+    "rimraf": "3.0.2",
+    "ts-jest": "25.3.0",
+    "ts-node": "8.8.1",
+    "typescript": "3.8.3"
   },
   "scripts": {
-    "test": "gulp tests",
-    "package": "gulp package"
+    "build": "rimraf build && echo Using TypeScript && tsc --version && tsc --pretty",
+    "clean": "rimraf build coverage",
+    "copy": "copyfiles -u 3 \"build/compiled/src/**/*\" build/package && copyfiles package.json README.md build/package",
+    "lint": "eslint --config ./.eslintrc.js --ext .ts ./src ./test",
+    "package": "npm run build && npm run copy && npm run public && rimraf build/compiled",
```

Comment:
I suggest to use https://github.com/mysticatea/npm-run-all here to make it cleaner.